code/modules/integrated_electronics/core/integrated_circuit.dm

```diff
@@ -287,12 +249,11 @@ a creative player the means to solve many problems.  Circuits are held inside an
 		update = FALSE
 
 	if(href_list["bottom"])
-		var/obj/item/integrated_circuit/circuit = locate(href_list["bottom"]) in src.assembly.contents
-		var/assy = circuit.assembly
+		var/obj/item/integrated_circuit/circuit = locate(href_list["bottom"]) in assembly.contents
```

Comment:
just `in assembly`

Reply:
I'm rewriting this completely in another PR.